src/plugins/console/public/application/containers/editor/monaco/utils/status_code_decoration_utils.test.ts

```diff
@@ -91,108 +95,45 @@ describe('getStatusCodeDecorations', () => {
     const EXPECTED_DECORATIONS = [
       {
         range: {
-          endColumn: 21,
+          endColumn: 1,
           endLineNumber: 1,
-          startColumn: 15,
+          startColumn: 1,
           startLineNumber: 1,
         },
         options: {
-          inlineClassName: SUCCESS_STATUS_BADGE_CLASSNAME,
+          isWholeLine: true,
+          blockClassName: SUCCESS_STATUS_CODE_CLASSNAME,
+          marginClassName: SUCCESS_STATUS_CODE_LINE_CLASSNAME,
         },
       },
       {
         range: {
-          endColumn: 28,
+          endColumn: 1,
           endLineNumber: 12,
-          startColumn: 13,
+          startColumn: 1,
           startLineNumber: 12,
         },
         options: {
-          inlineClassName: WARNING_STATUS_BADGE_CLASSNAME,
+          isWholeLine: true,
+          blockClassName: WARNING_STATUS_CODE_CLASSNAME,
+          marginClassName: WARNING_STATUS_CODE_LINE_CLASSNAME,
         },
       },
       {
         range: {
-          endColumn: 47,
+          endColumn: 1,
           endLineNumber: 18,
-          startColumn: 22,
+          startColumn: 1,
           startLineNumber: 18,
         },
         options: {
-          inlineClassName: DANGER_STATUS_BADGE_CLASSNAME,
+          isWholeLine: true,
+          blockClassName: DANGER_STATUS_CODE_CLASSNAME,
+          marginClassName: DANGER_STATUS_CODE_LINE_CLASSNAME,
         },
       },
     ];
 
     expect(getStatusCodeDecorations(SAMPLE_COMPLETE_DATA)).toEqual(EXPECTED_DECORATIONS);
   });
-
-  it('only returns decorations for data with complete status code and text', () => {
```

Comment:
Removing this test as now we have a decoration for the first line of every response regardless of the completeness of the status code and text